XCUITests/AuthenticationTest.swift

```diff
@@ -263,4 +265,24 @@ class AuthenticationTest: BaseTestCase {
         waitForExistence(app.navigationBars["Enter Passcode"])
         navigator.performAction(Action.UnlockLoginsSettings)
     }
+
+    func testBasicHTTPAuthenticationPromptVisible() {
+        navigator.openURL(testBasicHTTPAuthURL)
+
+        waitForExistence(app.staticTexts["Authentication required"])
```

Comment:
It may be good to add here a timeout: `waitForExistence(app.staticTexts["Authentication required"], timeout: 5)` since BB is usually slower than running tests locally and the Authentication dialog may take a bit to appear...

Reply:
In fact the test failed on BB due to that:
https://dashboard.buddybuild.com/apps/57bf25c0f096bc01001e21e0/build/5f3510632d12ba0001474025/test/d6975419-8696-42d7-b152-2392bcc509cd/9d19abb4-57c6-4989-ad1b-40faf07c287a